benches/pool.rs

```diff
@@ -2,17 +2,19 @@
 
 extern crate test;
 
-const CAPACITY: usize = 4_000; // 4kb
+const CAPACITY: usize = 4 * 1024; // 4kib
 
 mod remem {
     use super::CAPACITY;
     use remem::Pool;
     use std::thread;
-    use test::Bencher;
+    use test::{black_box, Bencher};
 
     #[bench]
     fn create(b: &mut Bencher) {
-        b.iter(|| Pool::<Vec<usize>>::new(|| vec![0; CAPACITY]));
+        let pool = Pool::<Vec<u8>>::new(|| vec![0; CAPACITY]);
+
+        b.iter(|| black_box(pool.get()));
```

Comment:
We should probably wrap the `Pool` in the black box here instead. We're now measuring the init cost + initializing a single vec. Not that this test is super important tho.